crates/weaver_forge/README.md

```diff
@@ -201,6 +215,44 @@ e.g. \[\[a,b\],\[c\]\] => \[a,b,c\]
 - `attribute_namespace`: Converts {namespace}.{attribute_id} to {namespace}.
 - `required`: Filters a list of `Attribute`s to include only the required attributes. The "conditionally_required" attributes are not returned by this filter.
 - `not_required`: Filters a list of `Attribute`s to only include non-required attributes. The "conditionally_required" attributes are returned by this filter.
+- `markdown_to_html`: Converts a markdown string to an HTML string.
+- `text_map`: Converts an input into a string based on the `text_maps` section of the `weaver.yaml` configuration file and a named text_map.
```

Comment:
Similarly to the ansi comment, should this be `weaver_text_map` so folks know it interacts with the weaver config?  I'm less certain of this one.

Reply:
Not sure about this one either. I have a preference for a shorter filter name in this context.

Reply:
Agreed. We can evaluate as we grow filters.